detection_rules/semver.py

```diff
@@ -30,6 +30,19 @@ def __str__(self):
 
         return recovered_str
 
+    def bump(self, major: bool = False, minor: bool = False, patch: bool = False) -> 'Version':
```

Comment:
Are there cons to removing our custom semantic versioning module and installing the one available from pypi? If there are no cons, we should maybe look to make this a separate issue to replace it. Just a thought.

Reply:
The major concern is in things we've seen in the past where our `Version` behaves differently than [semantic-version](https://pypi.org/project/semantic-version/). Our logic and assumptions to compare Versions I believe behaves differently at times.